test/lib/ufe/testContextOps.py

```diff
@@ -158,6 +180,16 @@ def shadingVariant():
 
     @unittest.skipIf(os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') < '2015', 'testAddNewPrim only available in UFE preview version 0.2.15 and greater')
     def testAddNewPrim(self):
+
+        # Create a proxy shape with empty stage to start with.
+        import mayaUsd_createStageWithNewLayer
+        mayaUsd_createStageWithNewLayer.createStageWithNewLayer()
+
+        # Create our UFE notification observer
+        ufeObs = TestAddPrimObserver()
+        ufe.Scene.addObjectDeleteObserver(ufeObs)
+        ufe.Scene.addObjectAddObserver(ufeObs)
```

Comment:
Use a UFE observer helper class to make sure when we add/remove prims we get the proper UFE add/delete notifs.